requirements/pip.txt

```diff
@@ -72,15 +72,15 @@ pytz==2019.1
 Unipath==1.1
 django-kombu==0.9.4
 mock==3.0.5
-stripe==2.27.0
+stripe==2.28.1
 
 # unicode-slugify==0.1.5 is not released on PyPI yet
 git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.5
 
 django-formtools==2.1
 django-crispy-forms==1.7.2
 
-docker==3.7.2
+docker==4.0.1
```

Comment:
I don't see anything that could break in our side https://docker-py.readthedocs.io/en/stable/change-log.htm but we should be careful when deploying :)

Reply:
I think it's fine. We are deploying with 3.7.2 as a test as well. We need to check that there is no timeout when executing commands in the Docker container. That is what was happening and why we pinned the previous version.

I'm noting this for the next deploy, just in case.